udata_front/theme/gouvfr/assets/js/index.ts

```diff
@@ -78,7 +78,7 @@ const configAndMountApp = (el: HTMLElement) => {
   app.component("user-reuse-list", UserReuseList);
 
   // @ts-ignore unset delimiters used in html templates to prevent injections using {{ }}
-  app.config.compilerOptions.delimiters = [];
+  app.config.compilerOptions.delimiters = ["[[[[[", "]]]]]"];
```

Comment:
I'm unsure about this. It may be unlikely to be found in legit user content but still leave room for a dedicated attacker